app/backend/approaches/chatreadretrieveread.py

```diff
@@ -196,7 +196,7 @@ async def run_until_final_call(
             # Azure Open AI takes the deployment name as the model name
             model=self.chatgpt_deployment if self.chatgpt_deployment else self.chatgpt_model,
             messages=messages,
-            temperature=overrides.get("temperature", 0.7),
+            temperature=overrides.get("temperature", 0.3),
```

Comment:
I could also make a class variable on approach class, but there's no precedent for that yet, so this is more straightforward change.

Reply:
Note that slider PR or this PR will require mods, depending which gets approved first.

Reply:
I think this is OK for now